libs/dmlf/src/networkers/muddle_learner_networker.cpp

```diff
@@ -75,10 +75,32 @@ MuddleLearnerNetworker::MuddleLearnerNetworker(const std::string &cloud_config,
   mud_->SetPeerSelectionMode(muddle::PeerSelectionMode::KADEMLIA);
 
   std::unordered_set<std::string> initial_peers;
-  if (instance_number > 0)
+  auto                            config_peers = doc.root()["peers"];
+
+  auto config_peer_count = config_peers.size();
+  for (std::size_t peer_number = 0; peer_number < config_peer_count; peer_number++)
+  {
+    if (peer_number != instance_number)
+    {
+      peers_.emplace_back(config_peers[peer_number]["pub"].As<std::string>());
+    }
+  }
+
+  if (config_peer_count < 10)
```

Comment:
Could we make 10 a parameter so that it's clear what this is doing? (I.e. replace the 10 below as well with something like min_peers)